.gitlab-ci.yml

```diff
@@ -7,6 +7,6 @@ Unit Test:
   script:
     - apt-get update
     - apt-get install -y git
-    - pip install -r requirements.txt
+    - pip install --upgrade pip
```

Comment:
yeah, because of this line in [requirements-dev.txt](https://github.com/ebi-ait/ingest-archiver/blob/be77048d6981a3780e7da4a68f090480b65ad1a4/requirements-dev.txt#L5)